src/main/323C.c

```diff
@@ -1434,7 +1436,20 @@ INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002AE90);
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002AF4C);
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002B008);
+struct QuadObj* FindFreeQuadObj()
```

Comment:
Nit: why not `find_free_quad_obj` to match the other `find` funcs? Should we update those to match?

Reply:
Yeah I've been indecisive on naming style and we've ended up with a mix. I suppose we ought to pick something. I'm contemplating the rust style? https://doc.rust-lang.org/1.0.0/style/style/naming/README.html

Basically in C terms  `snake_case` for functions, `CamelCase` for types, `SCREAMING_SNAKE_CASE` for constants, `snake_case` for variables?

Open to suggestions

Reply:
No strong preference, happy to align to whatever you choose!

Reply:
Ok let's go with that then, I don't think your PR needs any updating in that case, I'll adjust my name when combining the redundant QuadObjs